### PR TITLE
Docs for 0.1

### PR DESCRIPTION
**Pull request by MarioArias**

### CODE REVIEW

File: docs/rst/users/akka.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+####
+Akka
+####
+
+The Akka monitoring currently logs:
+
+* The number of messages received per actor type
```
Comment (janm399) [inline docs/rst/users/akka.rst]: Actor type or path. We support both filter expressions.

File: docs/rst/users/akka.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+* The number of actors of a given actor type
+* Actor queue sizes
+* Duration of an actor's response to receiving a message
+* Objects published by the EventStream
```
Comment (janm399) [inline docs/rst/users/akka.rst]: Code should always be monospaced.

File: docs/rst/users/akka.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* Invoked failure handling on an actor
```
Comment (janm399) [inline docs/rst/users/akka.rst]: Clumsy English

File: docs/rst/users/akka.rst [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+and can be configured to sample 1-in-n messages to a given actor type
+
+
+The org.eigengo.monitor.agent.AgentConfigurationFactory class searches for a mandatory configuration file at ``META-INF/monitor/agent.conf``. The configuration file must have this format::
```
Comment (janm399) [inline docs/rst/users/akka.rst]: Code should be monospaced.

File: docs/rst/users/akka.rst [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+            ]
+        }
+    }
+
+
+Parameters
+==========
+
+``output.class``
+    The class to be used for the output of the pointcut monitoring
```
Comment (janm399) [inline docs/rst/users/akka.rst]: ... a class that must contain nullary constructor.

File: docs/rst/users/akka.rst [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+``akka.includeRoutees``
+    If true, we tag routed actors with their own full actor path, as well as that of the parent
```
Comment (janm399) [inline docs/rst/users/akka.rst]: Doesn't make sense.

File: docs/rst/users/akka.rst [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+``akka.included``
+    The actor classes to monitor. We can match on class name or actor path. This list is currently treated as a conjunction -- i.e. an actor must match all filters to be included. This behaviour may change.
```
Comment (janm399) [inline docs/rst/users/akka.rst]: Let's leave out "this may change"

File: docs/rst/users/akka.rst [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+``akka.sampling``
+    Defines the rate at which to sample messages. This is optional -- default is to sample every message
+``akka.sampling.rate``
+    Integer value 'n' - sample every 'nth' message to an actor type (starting with the first)
```
Comment (janm399) [inline docs/rst/users/akka.rst]: Actor filter, not type

**Comment by janm399** [comment]
See if you can refer to `@version@` in the documentation rather than the literal `0.1-SNAPSHOT`?

**Comment by anirvanc** [comment]
LGTM :+1:

**Pull request merged**